src/test/java/core/basesyntax/service/impl/DailyReportFileReaderImplTest.java

```diff
@@ -0,0 +1,106 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.exceptions.EmptyFileException;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DailyReportFileReader;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class DailyReportFileReaderImplTest {
+    private static final Path TEST_FILE_ABSENT =
+            Path.of("src/test/resources/dailyOperationTest_AbsentFile.csv");
+    private static final Path TEST_FILE_EMPTY =
+            Path.of("src/test/resources/dailyOperationTest_EmptyFile.csv");
+    private static final Path TEST_FILE_PROPER_DATA_ONE_OPERATION =
+            Path.of("src/test/resources/dailyOperationTest_ProperDataOneOperation.csv");
+    private static final Path TEST_FILE_PROPER_DATA_THREE_OPERATION =
+            Path.of("src/test/resources/dailyOperationTest_ProperDataThreeOperation.csv");
+    private static final Path TEST_FILE_PROPER_DATA_SEVEN_OPERATION =
+            Path.of("src/test/resources/dailyOperationTest_ProperDataSevenOperation.csv");
+    private static final Path TEST_FILE_PROPER_DATA_TWELVE_OPERATION =
+            Path.of("src/test/resources/dailyOperationTest_ProperDataTwelveOperation.csv");
+    private static DailyReportFileReader dailyReportFileReader;
```

Comment:
I think you could leave only the checking of operation "Three"

Reply:
Yeah, seems ugly :). Maybe too much, but it's a pity to remove them.